modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaManager.java

```diff
@@ -926,6 +959,23 @@ public CompletableFuture<Void> stopAsync(ComponentContext componentContext) {
         shutdownAndAwaitTermination(executor, shutdownTimeoutSeconds, TimeUnit.SECONDS);
         shutdownAndAwaitTermination(replicasCreationExecutor, shutdownTimeoutSeconds, TimeUnit.SECONDS);
 
+        try {
+            CompletableFuture.allOf(raftClientsFutures.values().toArray(new CompletableFuture<?>[0]))
+                    .get(shutdownTimeoutSeconds, TimeUnit.SECONDS);
+
+            raftClientsFutures.forEach((raftNodeId, raftClientFuture) -> {
+                raftClientFuture.thenAccept(RaftGroupService::shutdown);
+
+                try {
+                    raftManager.stopRaftNode(raftNodeId);
```

Comment:
Exception on the one raftNodeId will stop processing of the whole chain

Reply:
Fixed with `AssertionError`